bindings/java/users.md

```diff
@@ -0,0 +1,4 @@
+# Users
+
+- [QuestDB](https://github.com/questdb/questdb): An open-source time-series database for high throughput ingestion and fast SQL queries with operational simplicity.
```

Comment:
I remember Questdb uses OpenDAL from Rust.

See - https://github.com/questdb/roadmap/issues/5#issuecomment-2185849711

Reply:
Got it, will update.